naxsi_src/naxsi_runtime.c

```diff
@@ -2600,6 +2619,7 @@ ngx_http_naxsi_body_parse(ngx_http_request_ctx_t*     ctx,
   ngx_chain_t* bb;
   u_char*      full_body;
   u_int        full_body_len;
+  u_char*      content_type_str;
```

Comment:
this might need to be `const u_char*` ?

Reply:
Will it work
content_type_str = r->headers_in.content_type->value.data ?